src/storage/__init__.py

```diff
@@ -113,6 +114,10 @@ def _upload(hoster, path):
         return
     logging.info("Uploaded file to: " + url)
 
+    # shorten URL as defined in config
+    url_shortener = get_url_shortener(config[general]["url_shortener"])
+    url = url_shortener.shorten(url)
```

Comment:
We log the upload URL in line 115. Maybe we should consider logging the shortened URL too for consistency?

Reply:
Makes sense. Will do.